exporter.go

```diff
@@ -81,10 +81,11 @@ func main() {
 		metricsPath = flag.String("telemetry.path", "/metrics", "URL path for surfacing collected metrics")
 
 		cephConfig = flag.String("ceph.config", "", "path to ceph config file")
+                cephUser = flag.String("ceph.user", "admin", "Ceph user to connect to cluster. The key file must be named ceph.client.<user>.keyring.")
```

Comment:
There seems to be a slight alignment issue here. Can you run this through `gofmt`?

https://github.com/digitalocean/ceph_exporter/blob/master/CONTRIBUTING.md is typically a good reference to run through in general for PRs.